veggieseasons/lib/main.dart

```diff
@@ -83,11 +91,122 @@ class _VeggieAppState extends State<VeggieApp> with RestorationMixin {
           create: (_) => Preferences()..load(),
         ),
       ],
-      child: CupertinoApp(
+      child: CupertinoApp.router(
         theme: Styles.veggieThemeData,
         debugShowCheckedModeBanner: false,
-        home: const HomeScreen(restorationId: 'home'),
         restorationScopeId: 'app',
+        routerConfig: GoRouter(
+          restorationScopeId: 'router',
+          initialLocation: '/list',
+          redirect: (context, state) {
+            if (state.path == '/') {
+              return '/list';
+            }
+            return null;
+          },
+          debugLogDiagnostics: true,
+          routes: [
+            ShellRoute(
+              builder: (context, state, child) {
+                return HomeScreen(
+                  restorationId: 'home',
+                  child: child,
```

Comment:
pass the `ShellRoute` child to the `HomeScreen`, will be displayed inside the `CupertinoTabScaffold`